Source/DataSources/BoundingSphereState.js

```diff
@@ -0,0 +1,34 @@
+/*global define*/
+define([
+        '../Core/freezeObject'
+    ], function(
+        freezeObject) {
+    "use strict";
+
+    /**
+     * The state of a BoundingSphere computation being performed by a {@link Visualizer}.
+     * @exports BoundingSphereState
+     */
+    var BoundingSphereState = {
```

Comment:
Do we need to introduce this new type?  Or could we just introduce something generic?